tests/test_ddk.py

```diff
@@ -97,6 +99,35 @@ def test_K96(self):
         for p in testp.keys():
             adf = self.modelJ1713.d_phase_d_param(self.toasJ1713, delay, p)
 
+    def test_sini_from_value(self):
+        self.modelJ1713.SINI.value = 0.9
+        with pytest.raises(ValueError):
+            self.modelJ1713.validate()
+
+    def test_sini_from_par(self):
+        temp_par_str = (
```

Comment:
Why not use a triple-quoted string here?

Reply:
I consider the triple-quoted string is part of docstrings. Does this implementation has potential risk?

Reply:
Triple quotes provide a way to include long strings of any kind without a need for `\n`, commas, numerous quotes, and so on. Using multiple short strings inside is just more awkward syntax for the same thing.

Reply:
Yeah, triple quoted strings are nice, and easy to copy/paste into since they don't have all the extra cruft.

Reply:
Just changed.